package.json

```diff
@@ -5,7 +5,12 @@
 	"main": "index.js",
 	"scripts": {
 		"test": "echo \"Error: no test specified\" && exit 1",
-		"start": "node index.js"
+		"build": "webpack",
+		"build-babel": "npm run build-babel-server && npm run build-babel-client",
+		"build-babel-server": "babel src/server --out-dir ./dist",
+		"build-babel-client": "babel src/client --copy-files --out-dir ./dist/public",
+		"start": "node ./dist/server.js",
```

Comment:
So it appears npm start runs both the client and the server at the same time. At least that's what's happening on my machine. Is this what we want? Should we have a package.json at the top level directory or one for each sub project?